homeassistant/components/knx/websocket.py

```diff
@@ -0,0 +1,106 @@
+"""KNX Websocket API."""
+from __future__ import annotations
+
+from collections.abc import Callable
+
+import voluptuous as vol
+from xknx.telegram import Telegram, TelegramDirection
+from xknx.telegram.apci import GroupValueRead, GroupValueResponse, GroupValueWrite
+
+from homeassistant.components import websocket_api
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN, AsyncMessageCallbackType, MessageCallbackType
+
+
+def register_websocket_api(hass: HomeAssistant) -> None:
+    """Register the KNX Websocket API."""
+    websocket_api.async_register_command(hass, ws_info)
+    websocket_api.async_register_command(hass, ws_subscribe_telegram)
+
+
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "knx/info",
+    }
+)
+@websocket_api.async_response
+async def ws_info(
```

Comment:
Doesn't need to be async.
```suggestion
@callback
def ws_info(
```